Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -489,6 +489,35 @@ def audit_conflicts
     EOS
   end
 
+  def audit_keg_only_style
```

Comment:
@DomT4 Cool. Just a thought that the lowercase check it might be worth outputting the first word?